CommandHelper/LocalPackages/commands/chars/Captain Britain (BB).yml

```diff
@@ -0,0 +1,24 @@
+class: Captain Britain (BB).yml
```

Comment:
classes are supposed to be all lower case no spaces

Is there more than one Captain Britain? Not that I see. If not, then there's no reason to have BB